build.gradle

```diff
@@ -11,7 +11,7 @@ import com.synopsys.integration.log.PrintStreamIntLogger
 buildscript {
     ext {
         springBootVersion = '2.1.5.RELEASE'
-        blackDuckCommonVersion = '45.0.0'
+        blackDuckCommonVersion = '45.0.1'
```

Comment:
What was needed in blackduck-common that required the upgrade?

Reply:
the re-release of all the libraries with a change to integration-common